src/pages/guide/css/preprocess.md

```diff
@@ -263,6 +263,49 @@ In the processed file, this results in the following:
  // Theme widgets
 ```
 
+## Trigger flag deploy styles for only enabled modules
+
+Default all css by default will build to final css no matter modules enabled or disabled. With this flag explain below allow developers able to control how we want css generate
+
+
+### Changing flag trigger deploy static contents
+
+1. Add inside to app/etc/env.php or app/etc/config.php. Add following line to the top array
+
+```php
```

Comment:
Please increase the indent of the code block 3 spaces so that it lines up under step 1.